### ISSUE TITLE
Error running testclient.sh "Exception in thread "main" java.lang.UnsatisfiedLinkError: no jFabricCommon in java.library.path"

### ISSUE THREAD

**Issue by stevenalexander** [Azure/generator-azuresfjava#11]
Using an unmodified generated Reliable Actor Service, the application generates and builds fine, but when attempting to run the `testclient.sh` (from tutorial documentation [here](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-create-your-first-linux-application-with-java#start-the-test-client-and-perform-a-failover)) it gives the following error:

```
++ dirname ./testclient.sh
+ BASEDIR=.
+ cd .
++ pwd
+ export LD_LIBRARY_PATH=:/Users/stevena/source/azure/app-fabric/example1-java/azurejava-example1/MyactorsvcTestClient/out/lib/lib:/opt/microsoft/servicefabric/bin/Fabric/Fabric.Code/
+ LD_LIBRARY_PATH=:/Users/stevena/source/azure/app-fabric/example1-java/azurejava-example1/MyactorsvcTestClient/out/lib/lib:/opt/microsoft/servicefabric/bin/Fabric/Fabric.Code/
+ java -Djava.library.path=:/Users/stevena/source/azure/app-fabric/example1-java/azurejava-example1/MyactorsvcTestClient/out/lib/lib:/opt/microsoft/servicefabric/bin/Fabric/Fabric.Code/ -jar out/lib/myactorsvc-test.jar
Exception in thread "main" java.lang.UnsatisfiedLinkError: no jFabricCommon in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at system.fabric.utility.LttngLogger.<clinit>(LttngLogger.java:113)
	at microsoft.servicefabric.services.remoting.builder.MethodDispatcherBuilder.<clinit>(MethodDispatcherBuilder.java:36)
	at microsoft.servicefabric.actors.remoting.builder.ActorCodeBuilder.<init>(ActorCodeBuilder.java:31)
	at microsoft.servicefabric.actors.remoting.builder.ActorCodeBuilder.<clinit>(ActorCodeBuilder.java:22)
	at microsoft.servicefabric.actors.remoting.client.FabricActorProxyFactory.createActorProxy(FabricActorProxyFactory.java:102)
	at microsoft.servicefabric.actors.client.ActorProxyBase.create(ActorProxyBase.java:268)
	at microsoft.servicefabric.actors.client.ActorProxyBase.create(ActorProxyBase.java:237)
	at reliableactor.test.MyactorsvcTestClient.main(MyactorsvcTestClient.java:19)
```

This is running from Mac OS, gradle builds fine and resolves deps without error.

**Comment by suhuruli** [comment]
@raunakpandya to investigate

**Comment by rapatchi** [comment]
Hi @stevenalexander ,

Sorry for  the delayed reply. I have fixed the issue in the latest generator of version 1.0.7 .
PR reference: https://github.com/Microsoft/generator-azuresfjava/pull/13 

I have also updated the above doc on what should be done in case of mac. 
Basically, the issue is that we need to execute test client  inside the container not from the host machine. So i updated the doc on how to copy the test client into the container in case of mac.

Closing this, Let me know if you are facing any issues here.
